TWLight/users/models.py

```diff
@@ -121,8 +121,6 @@ class Meta:
     )
     # Temporary field to track sending of project page 2021 email to prevent duplication in case of error.
     project_page_2021_notification_sent = models.BooleanField(default=False)
-    # Temporary field to track sending of proxy launch email to prevent duplication in case of error.
-    proxy_notification_sent = models.BooleanField(default=False)
```

Comment:
Since you deleted a field on a model, you need to run `python manage.py makemigrations` and `python manage.py migrate` to update the database.

Reply:
Our [documentation](https://github.com/WikipediaLibrary/TWLight/wiki/Branches) says migration files aren't required - does that need updating?

Reply:
@suecarmol Done !